procrastinate/postgres.py

```diff
@@ -31,6 +31,14 @@
   AND (%(task_name)s IS NULL OR task_name = %(task_name)s)
 """
 
+select_old_jobs_sql = """
+SELECT job.id, task_name, lock, args, scheduled_at, queue_name
+    FROM procrastinate_jobs job
+WHERE status IN %(statuses)s
+  AND started_at < NOW() - (%(nb_hours)s || 'HOUR')::INTERVAL
+  AND (%(queue)s IS NULL OR queue_name = %(queue)s)
+"""
```

Comment:
I think it may be the moment where we could introduce a more abstract way to make queries:

```
class BaseJobStore:
	def perform_sql(self, query:str, **arguments) -> Iterable[Dict[str, Any]]:
		...
	def select_sql(self, query:str, **arugments) -> Iterable[Dict[str, Any]]:
		...
```

Reply:
Hmm, let's do this in a separate PR.

Reply:
I've done it in #71 , now it's a race :D

Reply:
Hm, actually I've done a part of that in #71 but not the whole.